ros2_control_demo_bringup/config/rrbot_with_sensor_controllers.yaml

```
@@ -0,0 +1,25 @@
+controller_manager:
+  ros__parameters:
+    update_rate: 2  # Hz
```

According to #126. It is much nicer movement.

```suggestion
    update_rate: 100  # Hz
``